backend/state.py

```diff
@@ -165,6 +168,7 @@ def initialize(self, domain, objects, true_predicates, all_goals, special_effect
         self.predicates = predicates
         self.actions = self._build_actions(domain, objects)
         self.goal = all_goals
+        self.param_objs = param_objs
```

Comment:
Previously, the type for param_arg_dict was Dict[Object, Object]. This meant that in robotouille_input, I needed to know the parameter objects in the state to access the valid actions and return the correct arguments. However, I have since replaced all instances of this with Dict[Str, Object], so that now robotouille_input can now look at the input json, see which station should be clicked on (e.g. "s2") and return the correct arguments, without needing the parameter Object, since it can just access the string directly from the dictionary.